engine/src/main/resources/org/pentaho/di/trans/steps/pentahoreporting/messages/messages_en_US.properties

```diff
@@ -65,3 +65,6 @@ PentahoReportingOutputMeta.Injection.FIELDNAME=The field name providing the sour
 ReportExportTask.ERROR_0001_TARGET_EXISTS=Target-File exists, but cannot be removed.
 ReportExportTask.USER_EXPORT_COMPLETE=Export complete
 ReportExportTask.USER_EXPORT_FAILED=Export failed
+PentahoReportingOutputDialog.OutputFilenameField.Label=Output file name field
+PentahoReportingOutputDialog.InputFilenameField.Label=Report definition file name field
+PentahoReportingOutputDialog.UseValuesFromField.Label=Accept file name from filed?
```

Comment:
typo here: it should be field and not filed

Reply:
Done